.github/workflows/llvm-project-epoch-three.yml

```diff
@@ -0,0 +1,36 @@
+name: stage 3 (epoch 3)
```

Comment:
Alternatively, we could put this into a single workflow.  The workflows can have stages that depend on each other.  I would prefer that really.

Reply:
> Alternatively, we could put this into a single workflow.

I considered that (one workflow with a `needs` directive on stage 3 for stage 2). One issue ATM is that changes to stage3 will necessitate rebuilds of stage2.  Maybe once the whole pipeline is flushed out, we can focus on condensing the number of stages?

For now, I'd rather keep them separate. Once we have the whole end to end flushed out, we can start making simplifications.

> I think this might be better as llvm-project (epoch 3)?

Sure.

> We should probably update the other ones too.

Will do.

Reply:
> We should probably update the other ones too.

I did this in #23 to keep this PR focused on epoch 3.

Reply:
ok, I will rebase on a landed #23

Reply:
done PTAL